AdvancedTask/AdvancedTask/Steps/NotificationSteps.cs

```diff
@@ -0,0 +1,85 @@
+﻿using AdvancedTask.AssertHelpers;
+using AdvancedTask.Pages.Components.ProfileOverview;
+using AdvancedTask.Utilities;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace AdvancedTask.Steps
+{
+    public class NotificationSteps:BaseClass
+    {
+        NotificationComponent NotificationComponentObj;
+        ProfileTabComponents ProfileTabComponentsObj;
+        NotificationAssertion NotificationAssertionObj;
+        public NotificationSteps()
+        {
+            NotificationComponentObj = new NotificationComponent();
+            ProfileTabComponentsObj = new ProfileTabComponents();
+            NotificationAssertionObj = new NotificationAssertion();
+        }
+        public void SeeAllClicked()
+        {
+
+            ProfileTabComponentsObj.ClickNotificationTab();
+                
+            bool SeeAllButtonClicked = NotificationComponentObj.SelectSeeAll();
+            NotificationAssertionObj.AssertNotificationSeeAll();
+
+        }
+        public void LoadMoreClicked()
+        {
+            ProfileTabComponentsObj.ClickDashboard();
+               
+            bool LoadMoreButtonClicked = NotificationComponentObj.SelectLoadMore();
+            Thread.Sleep(3000);
+            NotificationAssertionObj.AssertLoadMore();
+        }
+        public void ShowLessClicked()
+        {
+
+            ProfileTabComponentsObj.ClickDashboard();
+            bool ShowLessButtonClicked = NotificationComponentObj.SelectShowLess();
+            Thread.Sleep(3000);
+           NotificationAssertionObj.AssertShowLess();
+            Console.WriteLine("Verification is successful.");
+
+        }
+        public void SelectAlllicked()
+        {
+            ProfileTabComponentsObj.ClickDashboard();
+            bool AllNotificationSelected = NotificationComponentObj.SelectSelectAll();
+            Thread.Sleep(3000);
+            NotificationAssertionObj.AssertSelectAll();
+            Console.WriteLine("Verification is successful.");
```

Comment:
please remove all Console write line bad practice to have them

Reply:
Done

Reply:
I still see this in pages for your next task please do not add  this